### PR TITLE
Adding optional parameter "bucket region name"

### PR DESCRIPTION
**Pull request by moebiuz**
To use s3 api on a bucket outside default region (us-west-1), region_name must be specifyed, in other case End Point must be used instead bucket name, which is not a great option in some cases.

### CODE REVIEW

**Comment by iphoting** [comment]
Would `us-east-1` be the better default S3 bucket region choice?
<http://www.bucketexplorer.com/documentation/amazon-s3--amazon-s3-buckets-and-regions.html>

**Comment by moebiuz** [comment]
You are right

**Comment by dlapiduz** [comment]
@moebiuz nice! looks cool

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ AWS_ACCESS_KEY_ID="your_key" \
 AWS_SECRET_ACCESS_KEY="your_secret" \
 letsencrypt --agree-tos -a letsencrypt-s3front:auth \
 --letsencrypt-s3front:auth-s3-bucket the_bucket \
+[ --letsencrypt-s3front:auth-s3-region your-bucket-region-name ] (default is us-west-1) \
```
Comment (iphoting) [inline README.md]: This should also read `us-east-1`. :)

Comment (moebiuz) [inline README.md]: Fixed. Thank you

**Pull request merged**